src/preamble.js

```diff
@@ -736,6 +738,7 @@ function instrumentWasmTableWithAbort() {
   var realGet = wasmTable.get;
   var wrapperCache = {};
   wasmTable.get = function(i) {
+    {{{ from64('i') }}}
```

Comment:
Are function pointers 64-bit in wasm64? I thought only the memory was.

Reply:
Yes they are, for compatibility with other pointer types.